rc/windowing/zellij.kak

```diff
@@ -0,0 +1,74 @@
+# https://zellij.dev/
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+provide-module zellij %{
+
+# ensure we're running under zellij
+evaluate-commands %sh{
+    [ -z "${kak_opt_windowing_modules}" ] || [ -n "$ZELLIJ" ] || echo 'fail zellij not detected'
+}
+
+define-command -hidden -params 2.. zellij %{
+    evaluate-commands %sh{
+       zellij "$@" < /dev/null > /dev/null 2>&1 &
+    }
+}
+define-command -hidden -params 1.. zellij-terminal-impl %{ evaluate-commands %sh{
+   arg_count=0
+   i=1
+   while [ $i -lt $# ]; do
+       a=${@:$i:1}
+       if [ "${a:0:1}" = "-" ]; then
+           if [ "${a:1:1}" = "d" -o "${a:1:1}" = "n" ]; then
+               ((i+=2))
+           else
+               ((i+=1))
+           fi
+           arg_count=$((i-1))
+       else
+           arg_count=$((i-1))
+           i=$#
+       fi
+   done
```

Comment:
this allow `:terminal` accept options like `-c` (close-on-exit) or `-d down`

Reply:
I dont think we should support that for now, `terminal` is a unified interface for various windowing provider. Additionally this seems to be using bash array accesses, which is not supported by posix shells, so this wont work on non-bash shells.

Reply:
Make sense, fixed, thanks!